docs/storage-backends.rst

```diff
@@ -10,8 +10,8 @@ The default graphite setup consists of:
 * A carbon daemon writing data to the database
 * Graphite-web reading and graphing data from the database
 
-It is possible to switch the storage layer to something different than
-Whisper to accommodate specific needs. The setup above would become:
+It is possible to use an alternate storage layer than the default, Whisper, in
```

Comment:
ah, I see.  I removed "other" because it seemed redundant to "alternate".

What about `It is possible to use an alternate storage layer rather than the default,
Whisper, in order to accommodate specific needs.` ?